app/controllers/pages_controller.rb

```diff
@@ -28,7 +28,8 @@ def contributors
+      {name: 'Darryl Dixon', image: '/assets/contributors/darryl_dixon.png', profile: "As the saying goes: \"You are your own worst enemy.\" Your experience regarding this quote may very, but remember: nothing happens without your brain. Mental health is an important topic of discussion, one that far too many people are afraid to be open about, and far too many don't believe that this is a legitimate concern, resulting in few people finding someone to simply vent to, let alone getting help. The goal of if me is to break that habit, to help others with mental illness find help and a to communicate with their loved ones about their issues.<br><br>Darryl Dixon is a rad cat that loves web development, is a generalist full-stack JavaScript developer, and loves to give back to the community in some way, anywhere.", location: 'Virginia, USA', link_name: 'PieceDigital.net', link: 'https://PieceDigital.net', social: 'globe'},
+      {name: 'Nagma Kapoor', image: '/assets/contributors/nagma_kapoor.png', profile: "I have a B.A in Neurobiology from Rutgers University and I am pursuing a 2nd BSc. in Computer Science from Ryerson University. Mental health is a personal topic but good mental health relies on support, and through my interdisciplinary background I hope to bring my unique perspective here. Mental illness has been minimized in my community as a South Asian. You hear stories of family or friends in whispers. Only within my generation onward, there has been a minor shift, and I hope to see it open further. Personally, I have learnt to silently cope and create methods to keep my trajectory towards success without alerting anyone. Which is why I understand the need for an open-source project like If-Me. Sometimes, you need support from accepting family or friends without the eyes of an entire community on you. I think If-Me provides us with a safe space that extends out from our own selves and into the few that we trust. Building a closed community, that is free and open to individuals from all backgrounds, is a necessity. Access to mental health and resources need to reach those who need it most, and having an open source platform has been vital for it.", location: 'Toronto, Canada', link_name: 'nagmak', link: 'https://nagmak.github.io/', social: 'globe'}
```

Comment:
Can you spell If-Me as if me? :P In general, I've noticed the spelling of our app hasn't been consistent lol.